addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -315,6 +315,12 @@ export class HtmlField extends Component {
             this.props.record.model.bus.trigger("FIELD_IS_DIRTY", this._isDirty())
         );
 
+        if (this.props.isCollaborative) {
+            this.wysiwyg.odooEditor.addEventListener("onExternalHistorySteps", () =>
+                this.props.record.model.bus.trigger("FIELD_IS_DIRTY", this._isDirty())
```

Comment:
Missing `;`

Reply:
Since I am not using the brackets notation of the function there is no need for `;`

Reply:
oh my bad, I never multiline this syntax so I didn't check properly ^^